filebeat/fileset/modules.go

```diff
@@ -44,15 +44,15 @@ func newModuleRegistry(modulesPath string,
 				fcfg = &defaultFilesetConfig
 			}
 
-			if fcfg.Enabled != nil && (*fcfg.Enabled) == false {
-				continue
-			}
-
 			fcfg, err = applyOverrides(fcfg, mcfg.Module, filesetName, overrides)
 			if err != nil {
 				return nil, fmt.Errorf("Error applying overrides on fileset %s/%s: %v", mcfg.Module, filesetName, err)
 			}
 
+			if fcfg.Enabled != nil && (*fcfg.Enabled) == false {
```

Comment:
This applies the overrides first, which allows an override to disable a fileset. It allows a CLI config like:

`-M system.audit.enabled=false`